analysis.js

```diff
@@ -1,37 +1,38 @@
-var esprima = require("esprima");
-var options = {tokens:true, tolerant: true, loc: true, range: true };
-var fs = require("fs");
+import esprima from 'esprima';
+import fs from 'fs';
 
-function main()
+const options = {tokens:true, tolerant: true, loc: true, range: true };
+
+const main = () =>
 {
-	var args = process.argv.slice(2);
+	let args = process.argv.slice(2);
 
-	if( args.length == 0 )
+	if( args.length === 0 )
 	{
-		args = ["analysis.js"];
+		args = ['analysis.js'];
 	}
-	var filePath = args[0];
+	let filePath = args[0];
```

Comment:
lots of variables getting changed from

Reply:
Necessary change since var variable should be at the top of function.